src/Generator/ClassGenerator.php

```diff
@@ -11,33 +11,33 @@
 
 use Zend\Code\Reflection\ClassReflection;
 
-class ClassGenerator extends AbstractGenerator
+class ClassGenerator extends AbstractGenerator implements TraitUsageInterface
```

Comment:
@Ocramius Please check out my comment https://github.com/zendframework/zend-code/pull/109#issuecomment-307087267
Please note that this class __has all methods__ from this interface and in doc comments of these methods we had:
`@inherit Zend\Code\Generator\TraitUsageInterface`
what I've changed to `@inheritDoc` (but this is fine only when class has 'parent class' or 'interface' with these methods)